src/js/live-tracker.js

```diff
@@ -185,6 +155,7 @@ class LiveTracker extends Component {
       return;
     }
     this.reset_();
+    this.trigger('liveedgechange');
```

Comment:
We not fire `liveedgechange` on stopTracking so components can update appropriately.

Reply:
makes sense